upup/pkg/fi/cloudup/awstasks/subnet.go

```diff
@@ -196,6 +205,22 @@ func (_ *Subnet) RenderAWS(t *awsup.AWSAPITarget, a, e, changes *Subnet) error {
 		}
 	}
 
+	if e.IPv6CIDR != nil && strings.HasPrefix(aws.StringValue(e.IPv6CIDR), "/") {
+		vpcIPv6CIDR, err := findVPCIPv6CIDR(t.Cloud, e.VPC.ID)
```

Comment:
There's no dependency on the `VPCAmazonIPv6CIDRBlock` task, so `RunTasks` might schedule the `VPCAmazonIPv6CIDRBlock` task in a later slot.

Reply:
There is no value in the dependency, as `VPCAmazonIPv6CIDRBlock` will not return the associated block.
This is the reason for using `NewTryAgainLaterError` in `Subnet.RenderAWS`.

Reply:
But unless `VPCAmazonIPv6CIDRBlock.Run()` is called, the IPv6 association won't ever arrive. If there's no dependency, `RunTasks` might schedule `VPCAmazonIPv6CIDRBlock` to a later tranche than `Subnet`.

Reply:
> The IPv6 association won't ever arriveThere is a dependency in VPC, so it will arrive.

There is a dependency in VPC, so it will arrive, sooner or later.

> RunTasks might schedule VPCAmazonIPv6CIDRBlock to a later tranche than Subnet.

This is why `NewTryAgainLaterError` waits for it to appear.

Reply:
It won't appear due to how `RunTasks` schedules tasks. If tasks `b` and `c` depend on `a`, `d` depends on `b`, and `e` depends on `c`, the scheduler effectively puts false dependencies of `d` depending on `c` and `e` depending on `b`.